marketplace/deployer_util/schema_values_common.py

```diff
@@ -61,6 +61,11 @@ def load_schema(parsed_args):
   return config_helper.Schema.load_yaml_file(parsed_args.schema_file)
 
 
+@memoize
+def load_schema_and_validate(parsed_args):
+  return config_helper.Schema.load_yaml_file(parsed_args.schema_file).validate()
```

Comment:
nit: re-use `load_schema`, avoiding memoizing the content twice.

Reply:
Ah, thanks for the suggestion! Fixed.